.github/actions/templates/deployModule/action.yml

```diff
@@ -49,12 +49,16 @@ runs:
       shell: pwsh
       run: |
         # Load used funtions
-        . ".github/actions/sharedScripts/Convert-TokensInFiles.ps1"
+        . ".github/actions/sharedScripts/Convert-TokensInFileList.ps1"
         $TokensReplaceWithObject = @(
-            @{ Replace = "<<subscriptionId>>"; With = '${{ inputs.subscriptionId }}' }
+            @{ Replace = "<<tenantId>>"; With = '${{ env.ARM_TENANT_ID }}' }
+            @{ Replace = "<<tenantId2>>"; With = '${{ env.ARM_TENANT_ID_2 }}' }
```

Comment:
Either this should be `ID2 }}` instead of `ID_2` }}, or we update the principal ID ref and use e.g. `ID_1 }}` there. In any case it should be aligned